src/CameraGroup.cpp

```diff
@@ -509,8 +503,6 @@ void CameraGroup::refineCameraGroupAndObjectsAndIntrinsics(
               std::shared_ptr<Object3D> object_3d_ptr =
                   it_obj3d_ptr->object_3d_.lock();
               if (object_3d_ptr) {
-                const std::vector<cv::Point3f> &obj_pts_3d =
```

Comment:
@rameau-fr `obj_pts_3d` is never used. Is it correct behavior?

Reply:
same